src/patternfly/components/Page/page.scss

```
@@ -236,35 +234,25 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
 
 // Header navigation
 .pf-c-page__header-nav {
-  grid-column: 1 / -1;
-  grid-row: 2 / 3;
+  align-self: stretch;
   min-width: 0;
+  padding-right: var(--pf-c-page__header-nav--PaddingRight);
+  padding-left: var(--pf-c-page__header-nav--PaddingLeft);
   background-color: var(--pf-c-page__header-nav--BackgroundColor);
-
-  > .pf-c-nav {
-    min-width: 0;
-
-    .pf-c-nav__scroll-button {
-      @media screen and (max-width: #{$pf-global--breakpoint--lg - 1}) {
-        top: var(--pf-c-page__header-nav--c-nav__scroll-button--lg--Top);
-        background-color: var(--pf-c-page__header-nav--c-nav__scroll-button--lg--BackgroundColor);
-      }
-
-      &:nth-of-type(1) {
-        left: var(--pf-c-page__header-nav--c-nav__scroll-button--nth-of-type-1--Left);
-      }
-    }
-  }
+  grid-column: 1 / -1;
+  grid-row: 2 / 3;
 
   @media screen and (min-width: $pf-global--breakpoint--lg) {
     grid-column: 2 / 3;
     grid-row: 1 / 2;
-    flex: 1;
-    align-self: flex-end;
-    order: initial;
-    width: auto;
-    margin-right: var(--pf-c-page__header-nav--lg--MarginRight);
-    margin-left: var(--pf-c-page__header-nav--lg--MarginLeft);
+  }
+
+  .pf-c-nav {
+    --pf-c-nav__scroll-button--disabled--Color: var(--pf-c-page__header--c-nav__scroll-button--disabled--Color);
```

Wouldn't these be `header__c-nav`?

```suggestion
    --pf-c-nav__scroll-button--disabled--Color: var(--pf-c-page__header__c-nav__scroll-button--disabled--Color);
```

---

I don't think so, per our discussion earlier today